prawcore/rate_limit.py

```diff
@@ -16,13 +16,13 @@ class RateLimiter(object):
 
     """
 
-    def __init__(self) -> None:
+    def __init__(self, window_size: int) -> None:
```

Comment:
Can you make this a mandatory keyword argument?

Reply:
Changed this